auth_proxy/services.py

```diff
@@ -69,13 +89,20 @@ def cb_grantsetter(self, client_id, code, request, *args, **kwargs):
         """ OAuth2Provider Grant setter.
         """
         expires = datetime.utcnow() + timedelta(seconds=100)
+        user = self.db.session.query(User).\
```

Comment:
Does a client *need* a user? http://flask-oauthlib.readthedocs.io/en/latest/oauth2.html suggests it doesn't.